sys/net/network_layer/ipv4/addr/ipv4_addr_to_str.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ */
+
+#include <stdlib.h>
+
+#include "net/ipv4/addr.h"
+
+/* based on inet_ntop4() by Paul Vixie */
+char *ipv4_addr_to_str(char *result, const ipv4_addr_t *addr, uint8_t result_len)
+{
+    uint8_t n = 0;
+    char *next = result;
+
+    if ((result == NULL) || (addr == NULL) || (result_len < IPV4_ADDR_MAX_STR_LEN)) {
```

Comment:
I'm wondering if the check for `result_len` makes any sense as it proves nothing regarding the length of `result`. And it isn't used for anything else like condition or loop control. So it may be removed completely?